### DISCUSSION TITLE
Take over maintenance?

### DISCUSSION THREAD

**Discussion by kribblo** [kribblo/node-ffmpeg-installer#52] [Q&A]
I'm sorry guys, there's been a lot of stuff happening and I haven't been keeping up, also I do not use this project myself anymore. I would be happy to invite more people to have write access here and on npmjs, if there are any takers. Obviously you could also just fork it if that feels better. But yeah, probably not going to give you the service you deserve because not using it myself and life stuff in combination.

Let me know? I seriously do not want to let anyone down but I don't even have the tools to update installed right now...

**Comment by deevus** [comment]
I'd be happy to help

**Reply by kribblo** [reply]
Good, I will send you some info!

**Comment by deevus** [comment]
Got it! Will take a look during the week.

**Reply by rxst** [reply]
@deevus Do you have permission for this project?
